workers/unity/Packages/com.improbable.gdk.core/Logging/ForwardingDispatcher.cs

```diff
@@ -15,7 +15,7 @@ public class ForwardingDispatcher : ILogDispatcher
 
         private bool inHandleLog;
 
-        public Connection Connection { get; set; }
+        public Worker Worker { get; set; }
```

Comment:
Discussed offline that this needs to be WorkerSystem.

Reply:
I really don't agree here however?
This entire thing has nothing todo with ecs, and the WorkerSystem is nothing but a helper to get to worker owned bits. So why not just link directly to the worker?
Originally the Worker constructor even set the connection on the logDispatched it was passed, now it sets the worker instead.

Reply:
agree with that. We might want to use the dispatcher in monobehaviours which should not be dependent on the `WorkerSystem`

Reply:
With the current impl - you wouldn't create a dispatcher in MB code, so whether it depends on `Worker` vs `WorkerSystem` doesn't really matter.

Having said that, I'm in the camp of `Worker` > `WorkerSystem`

Reply:
WorkerSystem is not the helper. Worker is a set of helper methods for creating a worker. We just haven't got round to making the ecs based worker bootstrap.

Reply:
If we make it worker now we will just have to change it later.

Reply:
The class definition of `Worker` indicates it has the data definition of a `Worker` though - I don't see why this would be `WorkerSystem` as you'd just end up doing: `WorkerSystem.Worker`

Reply:
right now the data definition is in both `WorkerSystem` and `Worker`, which makes the overall definition of `WorkerSystem` and `Worker` more confusing. 
Even if you say that `Worker` is a set of helper methods, you could just as well use `Worker` instead of `WorkerSystem` everywhere outside of the ECS. 

For now, to keep it consistent with the MB workflow as well, I would use `WorkerSystem`, but we need to revisit on how we want to use the worker in an ECS flow vs a MB flow and remove any reference from `WorkerSystem` for anything that is not directly ECS related